src/main/java/com/microsoft/graph/http/DefaultHttpProvider.java

```diff
@@ -341,6 +347,11 @@ public void run() {
             final boolean shouldLogVerbosely = logger.getLoggingLevel() == LoggerLevel.DEBUG;
             logger.logError("Graph service exception " + ex.getMessage(shouldLogVerbosely), ex);
             throw ex;
+        } catch (final UnsupportedEncodingException ex) {
+        	final ClientException clientException = new ClientException("Unsupported encoding",
+                    ex);
+            logger.logError("Unsupported encoding exception " + ex.getMessage(), ex);
```

Comment:
nit: Should probably be "Unsupported encoding problem: "  (with colon) and could apply to other `logError` calls in this class.